x/wasm/types/proposal.go

```diff
@@ -852,6 +852,17 @@ func (p PinCodesProposal) ValidateBasic() error {
 	if len(p.CodeIDs) == 0 {
 		return sdkerrors.Wrap(ErrEmpty, "code ids")
 	}
+	for _, num1 := range p.CodeIDs {
+		if num1 == 0 {
+			return sdkerrors.Wrap(ErrZero, "0 is not accepted")
```

Comment:
No need to add a custom error message when it is the same as the one defined on the type. How about:
```suggestion
			return ErrInvalid.Wrap("0 is not accepted")
```